packages/core/src/configuration.ts

```diff
@@ -501,6 +505,7 @@ export const charts = {
 	minWidth: 150,
 	widthBreak: 600,
 	marginForLegendTop: 40,
+	marginForChartTitle: 24,
```

Comment:
We need more readability and extendability here:
e.g. `Configuration.charts.title.marginBottom`

We'll also have to change things like `marginForLegendTop` down the road, that was inherited from the old charts codebase...